env.py

```diff
@@ -94,6 +95,13 @@ def get_env_vars() -> (
 Please enable it by merging this pull request \
 so that we can keep our dependencies up to date and secure."
 
+    commit_message = os.getenv("COMMIT_MESSAGE")
+    if commit_message:
+        if len(commit_message) > 65536:
+            raise ValueError("COMMIT_MESSAGE environment variable is too long")
```

Comment:
I couldn't find any documentation on a length limit here, so just copied what the `BODY` does. Alternatively, I can just remove the check.

Reply:
Can't find it either with a quick search. Sounds reasonable to me! 👍🏻